contracts/src/o1js-pack/readme.txt

```diff
@@ -0,0 +1 @@
+As a workaround to https://github.com/o1-labs/o1js/issues/1651 , we are copying the relevant code over from o1js-pack into this repo
```

Comment:
Is there a reason to have the README as a .txt instead of a .md?

Reply:
We're not making use of any md syntax here.  It's just a note to future developers about why the library is being copied over here instead of imported.

Reply:
yes, but github renders markdown nicer than txt.

Reply:
I see.  I don't really want this note to be considered like documentation or instructions for how to use o1js-pack, so it's actually a feature IMO.